certora/specs/SocialRecoveryModule.spec

```diff
@@ -257,3 +275,26 @@ rule confirmRecoveryCanAlwaysBeInitiatedByGuardian(env e, address guardian, addr
         currentContract.recoveryRequests[safeContract].newOwners.length == newOwners.length &&
         currentContract.recoveryRequests[safeContract].newOwners[index] == newOwners[index];
 }
+
+// This rule verifies that the finalization cannot happen if the recovery module is not enabled.
+// Exceptions are made for the case where the Safe has only one owner and the recovery is initiated
+// with zero new owners and zero as the new threshold.
```

Comment:
Hmm, from my understanding of this comment, this should be a bug that needs to be fixed...